openpgp/integration_tests/v2/testdata/test_vectors.json

```diff
@@ -16,12 +16,12 @@
         "publicKey": "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nxjMEYvTtQBYJKwYBBAHaRw8BAQdAxsNXLbrk5xOjpO24VhOMvQ0/F+JcyIkckMDH\nX3FIGxfNIkdvbGFuZyBHb3BoZXIgPGdvbGFuZ0BleGFtcGxlLm9yZz7CkAQTFggA\nOBYhBIVAcQ5rOaiHWV0gjgy9+ctDgkE3BQJi9O1AAhsDBQsJCAcCBhUKCQgLAgQW\nAgMBAh4BAheAAAoJEAy9+ctDgkE3nJ4BAIp2vLwHlJwI+t6/b5QHIwKyztJ27lGG\niYIWwd008twoAQDy6B6L7WjAexX7dwbBZbMqTqwtrxk3zuLdIq44an0NDM44BGL0\n7UASCisGAQQBl1UBBQEBB0DkK5s+nyXuWtVDbmHsfHlc3YLzFqdGeelyPMSigJoQ\nDQMBCAfCeAQYFggAIBYhBIVAcQ5rOaiHWV0gjgy9+ctDgkE3BQJi9O1AAhsMAAoJ\nEAy9+ctDgkE3L3cBAO5W+YP2IrKxH4quutLsDBqHqS1H77ais7kOoXmHvN8tAQDe\nuRX4OR2Dic1BItJGUEX+zoXbbi9pCUkV5/A5gk3xAQ==\n=OBcx\n-----END PGP PUBLIC KEY BLOCK-----"
     },
     {
-        "encryptedSignedMessage": "-----BEGIN PGP MESSAGE-----\n\nwXYDUzQX5WbZ2DUSAcdAL8N/KsIV0/O1DJTQ9ZX2gSet+z4fpHlNGS0md8GNF8++\n7pt+XXGykKLnXDxtbMOQJGRYGbL0svgwpoHwU8eRXgDdcMr4GQwGlAz1GGQqfVyu\nkLwA3PRXR2zMtVQ3EU1+H4zrKcizA49j0sBHAQbGxg1ZU+6nbGXBLYdLs6y4+WcO\nozq45Jucl9cABn8uX9P4hXnncCAnMpMn4DkCPdQ9L+sTLER3WmqLvu/jQ7gMiqgs\nPkvgmt/qtl1NhS6ghnqrRSg3rxAZC9cA2hWV2cmLX9/h7MeiCn/78FIfH5XWOxKW\nMN9zYgzaMZ3M12hbw3/ma/qtXeHc1TeJ7fYGfwDBHIEFUPHtj685XfztcYsPoBi7\nQK4E+RFrpfyzhoU5HYJGsuuQXZIXAQlN6qCM1s9QwVk3zDw4d+H9JJI7YN3FoBrV\n5O9T2aU8Vnnpk6t+sDDZuoW8Igd/OYJkZGspm9ExY0n1mOVEW28NecVtW2nbmcDn\nwLg=\n=A1LE\n-----END PGP MESSAGE--------",
-        "message": "test\u554f\u91cf\u9bae\u63a7\u5230\u6848\u9032\u5e73\n",
-        "name": "curve448",
+        "encryptedSignedMessage": "-----BEGIN PGP MESSAGE-----\n\nwWwD9k8c3Xy6wgEaoEnCrDZ+ywupbVzpxqwY7xi2kBwUS0AkddAksga5gHmf\nxPSQs4qSqG7AvhXuZvUSZe6GxR05oNspCRJd08O05QIL/fmTaF/zS/2mKM3S\nT+zZhnKuAGR+4pkW7014E3zfiKbSwIoBHbZzQeY+L3PxjNN3Fw0rXGvAf7hb\nxUd/7Qi7fN4Q26i+NjJzp7Hfwm8vqHyMRlhoS/ByhEQgfqNvDQtMeWFRW9wf\ndefXCD05zFr4NckvlI6v4FBubRD6AM3yykCHjHjH7l93UoAduL/s3h1URnzT\nzoHqhr5kvrXg6vjrcynTwsQsSfluWGy1ZXhYzA5/kNU4kEweLs4N51gC+eY3\nab0qlsO1vxDcqKlp87Diiug9Jisa0jAUiy+N5CHZwZj1vlM4J++u1AlWvr7q\nortUCQnKISDJdXywfKSF3709Wn0tR+70J2TbVJ/YPp9bgt9iRP7Y39p2MBrB\nUE1CNu2nOpkczvHl4PAyV1g5XHNRWjJ8g83WtYtOaWrYqrnJdyekVf1ut2xa\naLzmmy6yDo++tbGA7lal4VuIWN9MdtFFDaoRFUXHtMsnH1T9+3E=\n=wpjW\n-----END PGP MESSAGE-----\n",
+        "message": "test\u554f\u91cf\u9bae\u63a7\u5230\u6848\u9032\u5e73\r\n",
+        "name": "curve448v6",
```

Comment:
```suggestion
        "name": "curve448_crypto_refresh",
```

Reply:
Thanks fixed, renamed it to `curve448rfc9580`